Product/Production/Deploy/pom.xml

```diff
@@ -824,12 +823,15 @@
             </modules>
         </profile>
 
-        <!-- JBoss 7 -->
-        <profile>
-            <id>jboss7</id>
-            <modules>
-                <module>jboss7</module>
-            </modules>
-        </profile>
-    </profiles>
+        <!-- Glassfish -->
+     	<profile>
+            <id>glassfish</id>
+			<activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Since we don't want to break CI job for this PR, we will leave it as default. The next PR will remove this.